Venmo/src/main/java/com/braintreepayments/api/VenmoIsReadyToPayCallback.java

```diff
@@ -11,8 +11,7 @@
 public interface VenmoIsReadyToPayCallback {
 
     /**
-     * @param isReadyToPay true if Venmo is ready; false otherwise.
-     * @param error an exception that occurred while checking if Venmo is ready
+     * @param venmoReadinessResult true if Venmo is ready; false otherwise.
      */
-    void onResult(boolean isReadyToPay, @Nullable Exception error);
+    void onVenmoReadinessResult(@NonNull VenmoReadinessResult venmoReadinessResult);
```

Comment:
This PR changes all public callbacks to follow the result method naming pattern of `on<ObjectBeingReturned>Result`. It can provide some code clarity if merchants are creating multiple callbacks (rather than all having the same `onResult` method), but it also feels pretty verbose. What do you think?

Reply:
Yeah it feels a bit verbose but imo not _too_ bad. We do have `SEPADirectDebit` all over so not sure anything can top that with verbosity 🙃

Reply:
In most cases I assume it'll be reduced to a lambda `(venmoReadinessResult) -> {}`. At the same time, it does offer flexibility to merchants who aren't using lambdas.